### PR TITLE
JNCI edits for parity on Unity SDK docs

### PR DESCRIPTION
**Pull request by jamesneimanconsulting**
@picoxr @david-pico This is a new branch and PR just for ensuring parity on the Unity SDK docs. 

Notes:

I've also fixed a number of rst formatting warnings in chapter 8
I've also fixed the figure number in chapter 7
v2.7.9 updates that could affect documentation:
"Supported Unity linear color space": removed FAQs around this as suggested in GH feedback; support mentioned in section 8.4.11.
"VR Compositor Layers": covered in section 8.4.7.
"Added customization for application launch animation": covered in section 8.4.5.
"Supported Unity version 2019.1.x": mentioned in section 3.2
"Supported LWRP": covered in section 8.5
v2.7.8 updates that affect docs:
"Support foverendered rendering (FFR)": covered in section 8.4.6.
"Support Unity version 2018.3、2018.4": I seem to recall that mention of this was to be removed at some point, hence it's NOT currently mentioned in section 3.2
v2.7.7 updates that affect docs:
"UnitySDK provides full-screen display (16:9)": Not mentioned.covered in the documentation. SMEs would need to provide the details.
"Add controler 3Dof and 6Dof interfaces": Described in section 5.2.1.
"Support read and configure Render Texture Size": covered in section 9.4

### CODE REVIEW

**Comment by picoxr** [comment]
@jamesneimanconsulting @david-pico
Comments：
1. Please ignore version 2.7.7 & 2.7.8, and keep doc structure as well as chapter names consistent with GitHub master branch if applicable. Essential descriptions of features in 2.7.7 & 2.7.8 have been covered in latest version. There is no need to merge contents of 2.7.7 or 2.7.8 into master branch.
2. Remove FAQ question "How do you turn off the auto-connection to 3DoF when running games on mobile phones?" , we no longer support mobile phone products.

**Comment by jamesneimanconsulting** [comment]
I've updated the chapter names to match what was previously used in master and also to correct some capitalizations to use upper case for all major words. Note that there is one additional chapter not in master (9 - Auxilliary Tools).

- The question "How do you turn off the auto-connection to 3DoF when running games on mobile phones?" has been removed from chapter 10.
The "feature mentions" for 2.7.8/2.7.7 features were dealt with as follows:
 2.7.8:
- Removed section 8.4.6: Foveated Rendering
- 2.2: replaced mention of support "Unity 2017.4 to 2019.1" with just "Unity 2019.1"
- Other 2.7.8 items such as "2d transmission function" were left in as they were mentioned as part of a broader set of workflow steps, and the content already existed in the master branch. The team should decide if and how this information should be removed.
 2.7.7:
- 3/6 dof support - previously mentioned in section 5.2 as part of broader steps for head and stereo tracking (and content was in master branch already), so it was left in. The team should decide if and how this information should be removed.
- Removed section 9.4: Configuring the Render Text Size.

**Comment by jamesneimanconsulting** [comment]
@david-pico I've made the requested updates to this branch with an additional commit, based on the feedback received last week.

File: PicoVR Unity SDK/chapter_eight.txt [hunk 1]
```diff
@@ -163,22 +163,7 @@ Figure 8.6 Splash Screen Interface.
 
 After selecting the type, click **OK** to set the configuration; a log message indicating **Congratulations** appears when the configuration is complete. 
 
-
-8.4.6	Foveated Rendering
```
Comment (picoxr) [inline PicoVR Unity SDK/chapter_eight.txt]: Foveated Rendering should be preserved.

File: PicoVR Unity SDK/chapter_nine.txt [hunk 2]
```diff
@@ -39,20 +39,3 @@ Figure 9.4 Option to set the frame rate
 
 
 Disable **Use Default FPS** and then enter the desired frame rate.
-
-9.4 Configuring the Render Text Size
```
Comment (picoxr) [inline PicoVR Unity SDK/chapter_nine.txt]: Chapter 9.4 should be preserved

**Review by picoxr — COMMENTED** [review]
Please merge chapter8 & chapter9 as "Advanced Features" and keep sub-chapter consistent with online doc here: http://static.appstore.picovr.com/docs/sdk/en/index.html

**Comment by jamesneimanconsulting** [comment]
@picoxr @david-pico  I've resolved the issues, merged chapter 9 with chapter 8, and chapter 10 became chapter 9. I also fixed some heading level 1 issues in the table of contents.

**Review by picoxr — APPROVED** [review]
Current version can be acceptted except for several duplication and out-dated contents, we'll approve but please note to make further changes as the following:
1. Remove out-dated chapter 2.4, 8.9, 8.10
2. Remove duplicated chapter 8.12, 8.13

**Comment by jamesneimanconsulting** [comment]
@picoxr @david-pico: I've updated the two chapters as requested. Note that chapters 8.12 and 8.13 were not duplicates.

**Review by picoxr — APPROVED** [review]
@jamesneimanconsulting Chapter 8.12, Chapter 8.13 contents are included in Chapter 4.3 & Chapter 4.4. Also note that Chapter 4.4 is marked as 4.3 by mistake.

**Pull request closed without merging**